schematic/store/synapse.py

```diff
@@ -3024,6 +3036,8 @@ def _fix_int_columns(self):
         for col in int_columns:
             # Coercing to string because NaN is a floating point value
             # and cannot exist alongside integers in a column
-            to_int_fn = lambda x: "" if np.isnan(x) else str(int(x))
+            def to_int_fn(x):
```

Comment:
when you define small in-line function like this, do you also add tests to this function?